### ISSUE TITLE
chat bubble not showing

### ISSUE THREAD

**Issue by maxchen-000** [hubtype/botonic#2648]
I have some action that fetch some data from the backend api which take few seconds. While it's fetching the data, there's no chat bubble that show the user that botonic is doing something. So it's hard to tell from user perspective whether the chat is not working or it's doing something. Is there some setting that I need to set to turn the chat bubble or this is a bug? The example on the botonic site does show the chat bubble.

Basically, it's missing this: 
![image](https://github.com/hubtype/botonic/assets/81391469/81aa331d-a472-4f2f-a33e-d9aa76469036)

**Comment by maxchen-000** [comment]
Added the following to the botonicInit() function to fix the issue:
`
Botonic.addBotText("Please wait, this might take some time...")
Botonic.setTyping(true)

// api call code
`
